client-app/shared/account/components/order-status.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <VcBadge :variant="orderStatus?.variant" :color="orderStatus?.color || defaultColor" truncate>
+  <VcChip :variant="orderStatus?.variant" :color="orderStatus?.color || defaultColor" truncate rounded>
```

Comment:
Please look at `IOrderStatus`, create and replace required typings. Also, the same statuses are used in quotes, what about them?